CHANGELOG.md

```diff
@@ -1,25 +1,41 @@
 # Changelog
 
-## [Unreleased](https://github.com/omgnetwork/elixir-omg/tree/HEAD)
+## [v1.0.5](https://github.com/omgnetwork/elixir-omg/tree/v1.0.5) (2020-10-01)
 
-[Full Changelog](https://github.com/omgnetwork/elixir-omg/compare/v1.0.4...HEAD)
+[Full Changelog](https://github.com/omgnetwork/elixir-omg/compare/v1.0.4-pre.2...v1.0.5)
```

Comment:
Shouldn't we compare with the last released release? `1.0.3` instead of `v1.0.4`?

Reply:
I don't think that is a good idea. We have changelog for 1.0.4 already and 1.0.4 is a stable release (as it passed sandbox) just it did not reach production. Compare to 1.0.3 might cause more confusion IMO.